data-raw/R2R_TMH_habitat_inputs/documentation/TMH_methodology.Rmd

```diff
@@ -0,0 +1,178 @@
+The Combined Habitat Suitability raster[^3] was aggregated by suitability score and the North and South Deltas. It was found that combined habitat suitability scores 50 and greater matched the average existing SIT modeled Delta habitat. The North Delta has 2767 acres with rearing habitat score of 50 and greater, the South Delta has 7971 acres.
+
+[^3]: Looking at individual habitat parameters separately is of limited value because criteria for multiple habitat parameters must be met in order for an area to truly provide suitable rearing habitat. However, considering suitability across parameters is challenging because 1) suitability has not been determined for all important parameters, and 2) there is uncertainty around which habitat parameters are most important, and whether there are interactions between parameters, both of which are important for determining how criteria for individual parameters should be combined. The combined suitability map presented here was created by taking the geometric mean of suitability scores for the four parameters for which suitability was assessed: weighted wetland edge, wetland adjacency, water depth, and water temperature. Numeric weights were given to reflect relative suitability for each of the criteria. Suitability values ranged from 0 (unsuitable) to 100 (most suitable). Water depth was the only criteria that had values weighted zero. This combined map is meant to be an initial look at how parameters might be combined and is expected to be refined in future efforts as data availability and our understanding of salmon rearing habitat in the Delta increases. <https://www.sfei.org/data/delta-salmon-rearing-geospatial-analyses#sthash.KKeTs6Gx.dpbs>
+
+#### Identify Delta Landscapes Usable in Maximum Habitat
+
+We defined SFEI's "restorable" landscapes as the minimally subsided and intertidal areas[^4] (Figure 1, Table X).
+
+[^4]: <https://www.sfei.org/projects/delta-landscapes-scenario-planning-tool>
+
+Table X. Minimally Subisded and Intertidal areas with SFEI combined habitat suitability \>50 removed
+
+| **Delta** |    **Dataset**     | **Acreage** |
+|:---------:|:------------------:|:-----------:|
+|   North   | Minimally Subsided | **31,369**  |
+|   North   |     Intertidal     | **10,351**  |
+|   South   | Minimally Subsided | **78,122**  |
+|   South   |     Intertidal     | **24,669**  |
+
+[![](images/SFEI_max_hab.png){width="548"}](https://deltaplanningmap.sfei.org/)
+
+Figure 1. SFEI mapped minimally subsided and intertidal areas in the North and South Deltas
+
+### Limitations
+
+-   Most watersheds are modeled using GIS methodology and therefore results are divorced from instream flows 
+
+-   Modeling assumes a linear riverine system
+
+-   Assumes salmonid migration past potential barriers, such as dams
+
+-   In-channel and floodplain flows are not considered in the proportion suitable metric 
+
+-   Assumes natural systems in areas that have been heavily impacted by urbanization \
+    \
```

Comment:
whats up with the \

Reply:
I have no idea... it wasn't visible in the HTML. removed though.